cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -73,6 +73,8 @@ message SeismicStore {
     IngestionSource ingestion_source = 4; // How the file was ingested
     map<string, string> metadata = 5;
     File ingested_file = 6; // The info for the original file, if it was ingested as a file
+    VolumeDef inline_volume_def = 7;
```

Comment:
Add some comments to indicate what these VoluemDef objects are about, and the difference between these two fields, and I am happy to merge this. =)

Reply:
I've added a comment on the design doc about having crosslines as the keys in the volume defs. If we don't want to use crossline values as keys then I'll remove the `crossline_volume_def` in this message and the column in the metadata store